templates/index.html

```diff
@@ -12,90 +12,145 @@
-  <section class="full half-1-md half-1-lg">
+<!-- <nav class="full-bleed full grid12"> -->
+
+<section class="full c2-7-md c3-7-lg mt6">
+  <p class="lh-copy mt1 mb4">
+    It's a simple, durable computer you own 100% and forever.
+  </p>
+</section>
 
-    <ul>
-      <li><a href="/understanding-urbit/">Introduction -></a></li>
+<section class="full c2-7-md c3-7-lg mt6">
+  <ul class="list">
+    <li class="">
+      <a class="bb b--black" href="/understanding-urbit/">Introduction</a>
+    </li>
+    <li class="mt3">
+      <a class="blue2 bb b--blue" href="/docs">Docs</a>
+    </li>
+    <li class="mt3">
+      <a class="green2 bb b--green" href="/install">Install</a>
+    </li>
+  </ul>
+</section>
 
-      <li class="mt4"><a href="/essays">Essays</a></li>
-      <li><a href="/updates/">Updates</a></li>
-      <li><a href="/media">Media</a></li>
-      <li><a href="/faq/">FAQ</a></li>
+<section class="full c2-7-md c3-7-lg mt6">
+  <h1 class="mb3">Understanding Urbit</h1>
+    <p class="lh-copy mt1">This sequence of posts will give you a high-level overview of Urbit. They take about <code class="pv1 bg-gray4">~12m</code> to read.</p>
+  <ul class="list">
+    <li class="mt3">
+      <a class="b--black bb" href="/understanding-urbit/">Introduction</a>
+    </li>
+    {% set posts = get_section(path="understanding-urbit/_index.md") %} {%
+    for page in posts.pages %}
+    <li class="mt3">
+      <a
+        class="b--black bb"
+        href="{{ page.permalink }}">
+        {{ page.title }}
+      </a>
+    </li>
+    {% endfor %}
+  </ul>
+</section>
 
-      <li class="mt4"><a class="blue2" href="/docs">Docs ↗</a></li>
-      <li><a class="green2" href="/install">Install ↗</a></li>
-    </ul>
+<section class="full c7-12-md c7-11-lg mt6">
+  <h1 class="mb3">Learn More</h1>
+  <ul class="list">
+    <li class="mt3">
+      <a class="b--black bb" href="/essays">Essays</a>
+    </li>
+    <li class="mt3">
+      <a class="b--black bb" href="/updates/">Updates</a>
+    </li>
+    <li class="mt3">
+      <a class="b--black bb" href="/media">Media</a>
+    </li>
+    <li class="mt3">
+      <a class="b--black bb" href="/faq/">FAQ</a>
+    </li>
+  </ul>
+</section>
 
-  </section>
+<section class="full c2-7-md c3-7-lg mv6">
+  <h1 class="mb3">Follow Us</h1>
+  <p class="lh-copy mt1 mb4">Urbit is actively being built, explored, and experimented with.
+    To follow our progress you can:
+  </p>
+  <ul class="list">
+    <li class="mt2">
+      Follow us on <a class="b--black bb" href="https://twitter.com/urbit">Twitter</a>
+    </li>
+    <li class="mt2">
+      Explore the code on
+      <a class="b--black bb" href="https://github.com/urbit/urbit">GitHub</a>
+    </li>
+    <li class="mt2">
+      Ask questions in our <a class="b--black bb" href="https://discord.gg/bkEX6EU">Discord</a>
+    </li>
+  </ul>
+  <p class="lh-copy">
+    or boot Urbit and join <code class="dt bg-gray4">~dopzod/urbit-help</code>
```

Comment:
It seems inconsistent to have a gray background on monospace just on the homepage. If it's a core aspect of code blocks maybe `@extend .bg-gray4` in `code, .code, .mono` in _main.scss. Personally I'm very partial toward no gray bg.

Reply:
Sure, not opinionated abt this